test/remote/gateways/remote_rapyd_test.rb

```diff
@@ -261,7 +261,7 @@ def test_failed_void_with_payment_method_error
   def test_failed_void
     response = @gateway.void('')
     assert_failure response
-    assert_equal 'UNAUTHORIZED_API_CALL', response.message
+    assert_equal 'NOT_FOUND', response.message
```

Comment:
I did this small fix in my previous [PR](https://github.com/activemerchant/active_merchant/pull/5018/files#diff-a5ef05dc1758ae76e6c577437c9e79d434d7ff58bac8276d8c416e5a415684e0R277) with other ones fixes, if you agree you can remove it from it.